tests/server_send.cpp

```diff
@@ -277,6 +277,42 @@ static void ssend_test_server_send(session &s, int num, const std::string &id_pr
 	BOOST_REQUIRE_EQUAL(copied, num);
 }
 
+#if (DISABLE_LONG_TEST)
+static void ssend_test_set_delay(session &s, const std::vector<int> &groups, uint64_t delay) {
+	struct backend {
+		dnet_addr addr;
+		int backend_id;
+
+		bool operator<(const backend &other) const {
+			if (auto cmp = dnet_addr_cmp(&addr, &other.addr))
+				return cmp < 0;
+			return backend_id < other.backend_id;
+		}
+	};
+
+	std::set<backend> backends;
+
+	for (const auto &route: s.get_routes()) {
```

Comment:
It builds fine on precise and, as I know, we do not support anything older than precise, do we?